packages/manager/modules/bm-server-components/src/general-information/installation/ovh/server-installation-ovh.controller.js

```diff
@@ -2226,7 +2231,7 @@ export default class ServerInstallationOvhCtrl {
     } else {
       storage.partitioning.schemeName = this.$scope.installation.selectPartitionScheme;
     }
-    this.$scope.installation.storage.push(storage);
+    this.$scope.installation.storage = [storage];
```

Comment:
Is that change part of the fix? What does it do exactly?

Reply:
No it's not, just that QA will not be able to test it because it occurs when API call fails.
I have found the issue while testing locally, because I voluntarily made the API call crashing and I noticed that once the error occurs, when user clicks again reinstall, it duplicates and pushes a new item to the storage array.
As reminder, storage array only accepts 1 single item for the moment.